src/load_balancer_stack.py

```diff
@@ -21,6 +22,50 @@ def __init__(
         self.alb = elbv2.ApplicationLoadBalancer(
             self, "AppLoadBalancer", vpc=vpc, internet_facing=True
         )
+
+        # WAF to protect against common web attacks (OWASP Top 10)
+        web_acl = wafv2.CfnWebACL(
+            self,
+            "WebAcl",
+            name="AppWebAcl",
+            default_action=wafv2.CfnWebACL.DefaultActionProperty(
+                allow=wafv2.CfnWebACL.AllowActionProperty()
+            ),
+            scope="REGIONAL",
+            visibility_config=wafv2.CfnWebACL.VisibilityConfigProperty(
+                cloud_watch_metrics_enabled=True,
+                metric_name="AWSManagedRulesCommonRuleSet",
+                sampled_requests_enabled=True,
+            ),
+            rules=[
+                wafv2.CfnWebACL.RuleProperty(
+                    name="AWSManagedRulesCommonRuleSet",
```

Comment:
You might also want to add [`AWSManagedRulesKnownBadInputsRuleSet`](https://docs.aws.amazon.com/waf/latest/developerguide/aws-managed-rule-groups-baseline.html#aws-managed-rule-groups-baseline-known-bad-inputs)

Reply:
@ConsoleCatzirl why do you recommend this one specifically and not any of the other ones on the managed rules list?

Reply:
@zaro0508 the main reason I recommend that rule group is for the `ExploitablePaths_URIPATH` rule because it gets a lot of hits on the prod Bridge WAF from constant bot scans.

Reply:
hmm, what if we add all of the AWS managed rules?  your thoughts?

Reply:
The third baseline group blocks access to admin APIs (e.g. anything with `admin` in the path) so it might not be appropriate depending on the use case. For example, blocking access to `/admin/login` would prevent admin logins to the Bridge server, so it can't be used there.

It would be more secure to only allow admin logins from a separate API Gateway that could only be accessed from the VPN, but since we only have a single public endpoint, it's not feasible.

Reply:
add AWSManagedRulesKnownBadInputsRuleSet